src/hermezjs-typings.d.ts

```diff
@@ -38,150 +34,156 @@ declare module "@hermeznetwork/*" {
     name: string;
     symbol: string;
     USD: number;
+    // fiatUpdate: ISOStringDate;
   };
 
-  export interface MerkleProof {
+  // type L1Info = {
+  //   amountSuccess: boolean;
+  //   depositAmount: string;
+  //   depositAmountSuccess: boolean;
+  //   ethereumBlockNum: number;
+  //   historicDepositAmountUSD: number;
+  //   toForgeL1TransactionsNum: number;
+  //   userOrigin: boolean;
+  // };
+
+  // type L2Info = {
+  //   fee: number;
+  //   historicFeeUSD: number;
+  //   nonce: number;
+  // };
+
+  export type MerkleProof = {
     root: string;
-    siblings: BigNumber[];
-    // oldKey: string;
-    // oldValue: string;
+    siblings: string[];
+    // fnc: number;
     // isOld0: boolean;
     // key: string;
+    // oldKey: string;
+    // oldValue: string;
     // value: string;
-    // fnc: number;
-  }
+  };
 
   export type Exit = HermezApiResourceItem & {
-    batchNum: number;
     accountIndex: string;
+    batchNum: number;
+    delayedWithdraw: unknown | null;
+    instantWithdraw: number | null;
+    // balance: string;
     // bjj: string;
+    // delayedWithdrawRequest: unknown | null;
+    // fee: number;
+    // hash: string;
     // hezEthereumAddress: string;
     // merkleProof: MerkleProof;
-    // balance: string;
-    instantWithdraw: number | null;
-    // delayedWithdrawRequest: unknown | null;
-    delayedWithdraw: unknown | null;
     // token: Token;
   };
 
-  export interface Withdraw {
+  export type PendingWithdraw = {
     accountIndex: string;
-    // amount: string;
     batchNum: number;
     hash: string;
-    // hermezEthereumAddress: string;
     id: string;
     timestamp: ISOStringDate;
+    // amount: string;
+    // hermezEthereumAddress: string;
     // token: Token;
-  }
-
-  export type DelayedWithdraw = Withdraw & {
-    // merkleProof: MerkleProof;
-    instant: boolean;
   };
 
-  export interface Deposit {
+  export type PendingDeposit = {
```

Comment:
All the `pending` interfaces don't belong to `hermezjs` but to our domain, as they are only used in the wallet. This concepts don't exist in the SDK.